library/ZendService/Apple/Apns/Client/AbstractClient.php

```diff
@@ -146,15 +146,17 @@ public function isConnected()
      * @param  int    $length
      * @return string
      */
-    protected function read($length = 1024)
+    protected function read($length = 6)
```

Comment:
@mwillbanks 
A response of APNs modified it for 6 bytes.

Please see below(Figure 5 - 2).

https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/CommunicatingWIthAPS.html